bin/main

```diff
@@ -1,3 +1,51 @@
 #!/usr/bin/env ruby
 
-puts "Hello World!"
\ No newline at end of file
+
+while true
+    puts "Player 1 choose between X and O"
+    player_one = gets.chomp
+
+    puts "1 | 2 | 3"
+    puts "4 | 5 | 6"
+    puts "7 | 8 | 9"
+    puts "Type the number of the box you want to put your #{player_one}"
+    player_one = gets.chomp
+    puts "Your move is diplayed on the board"
+    puts "X | 2 | 3"
+    puts "4 | 5 | 6"
+    puts "7 | 8 | 9"
+    puts "Player 2 choose the box you want"
+    player_two = gets.chomp
+    puts "Your move is diplayed on the board"
+    puts "X | 2 | 3"
+    puts "4 | O | 6"
+    puts "7 | 8 | 9"
+    puts "Player 1 choose the box you want"
+    player_one = gets.chomp
+    puts "Your move is diplayed on the board"
+    puts "X | X | 3"
+    puts "4 | O | 6"
+    puts "7 | 8 | 9"
+    puts "Player 2 choose the box you want"
+    player_two = gets.chomp
+    puts "Your move is diplayed on the board"
+    puts "X | X | O"
+    puts "4 | O | 6"
+    puts "7 | 8 | 9"
+    puts "Player 1 choose the box you want"
+    player_one = gets.chomp
+    puts "Your move is diplayed on the board"
+    puts "X | X | O"
+    puts "X | O | 6"
+    puts "7 | 8 | 9"
+    puts "Player 2 choose the box you want"
+    player_two = gets.chomp
+    puts "Your move is diplayed on the board"
+    puts "X | X | O"
+    puts "X | O | 6"
+    puts "O | 8 | 9"
+
+    puts "Player 2 wins"
+    puts "Do you want to play again? Y/N"
+    break
+end
```

Comment:
**[ OPTIONAL ]** Also this file shows a lot of errors with rubocop locally, and stickler does not pick it up because the file does not have the *.rb extension, which is ok, but in further milestones your are going to be required to fix those even though stickler is not picking them up.